pandas/util/testing.py

```diff
@@ -1324,11 +1323,13 @@ def assert_series_equal(left, right, check_dtype=True,
         # .values is an ndarray, but ._values is the ExtensionArray.
         # TODO: Use .array
         assert is_extension_array_dtype(right.dtype)
-        return assert_extension_array_equal(left._values, right._values)
+        assert_extension_array_equal(left._values, right._values)
+        return
```

Comment:
Hmm...not sure if this is ideal, but at least it makes it clear `assert_extension_array_equal` doesn't return anything.  Could potentially revisit in a follow-up.

Reply:
Is there actually a reason to return here? Seems like it would unintentionally skip out on subsequent checks

Reply:
@WillAyd : Not sure, but let's revisit in a follow-up, since the goal is to just refactor.

Reply:
Yea sure makes sense @samuelsinayoko if you'd like to tackle

Reply:
you can leave out the return statement. assert returns None which is the same